mkdocs.yml

```diff
@@ -25,7 +25,6 @@ markdown_extensions:
 
 plugins:
   - search
-  - git-revision-date-localized
```

Comment:
Just checking: mkdocs plugin was removed, please confirm this is desirable change.

Reply:
yeah, that should have gone to the "unstaged" changes. I will exclude it